src/index.d.ts

```diff
@@ -98,19 +104,29 @@ type ConnectData = {
 }
 
 type TransactionsOptions = {
-  institutionId: string;
+  institutions?: Array<string>;
   from?: string;
   to?: string;
   investments?: string;
   last?: string;
   limit?: number;
+  institutionId?: string;
 }
 
 type TransactionOptions = {
-  institutionId: string;
   txId: string;
 }
 
+type PositionsOptions = {
+  institutions?: Array<string>;
+  groupIds?: Array<string>;
+  institutionIds?: Array<string>;
+  investmentIds?: Array<string>;
+  includeAssets?: Boolean;
```

Comment:
Should we change it to just `assets` to match the API? Also because in the API, `?assets` can be either boolean or array of object ids like with `?institutions` and `?investments`.

And for consistency, maybe `includeLiabilities` should just be `liabilities` too

Reply:
Changing to identical to API for consistency